ffi/lib/dart_vlc_ffi.dart

```diff
@@ -6,19 +6,19 @@ import 'source/dynamiclibrary.dart';
 
 abstract class PlayerFFI {
```

Comment:
This file isn't actually part of plugin but my yesterday's research for future of this plugin.
Don't really mind changing of identifier names, but its kinda apart from plugin.

Reply:
I changed the identifier names because I was getting a lot of warnings because it wasn't `UpperCamelCase`. If you want, I'll leave it the way it was.

Reply:
It's alright.